generic-update-script.rb

```diff
@@ -131,13 +139,20 @@
   #########################################
   # Get update details for the dependency #
   #########################################
-    checker = Dependabot::UpdateCheckers.for_package_manager(package_manager).new(
+  checker = Dependabot::UpdateCheckers.for_package_manager(package_manager).new(
     dependency: dep,
     dependency_files: files,
     credentials: credentials,
   )
-  next unless dep.display_name.match(/^Agoda/) 
-  next if checker.up_to_date?
+  unless dep.display_name.match(/^Agoda/) then
```

Comment:
#1 we need to change this to address this issue, will there be anohter PR for this?

Reply:
yep, separate issue

Reply:
pushed within the same pr to avoid conflicts